utils/bazel/llvm-project-overlay/llvm/BUILD.bazel

```diff
@@ -297,7 +320,21 @@ cc_library(
     }),
     includes = ["include"],
     linkopts = select({
-        "@platforms//os:windows": [
+        ":is_windows_clang_mingw": [
+            "-lole32",
+            "-luuid",
+            "-lws2_32",
+            "-lntdll",
+        ],
+        ":is_windows_clang_cl": [
+            "ole32.lib",
+            "uuid.lib",
+            "ws2_32.lib",
+            "ntdll.lib",
+        ],
+        ":is_windows_msvc": [
```

Comment:
Are there other windows compilers besides clang/msvc to support? If so, do we need a windows default?

Reply:
There won't be an analysis error since there is a default below, but it's unix-y: `-pthread -ldl -lm`. Hopefully that's will lead to a strange enough error that it will be obvious if something goes wrong. This is probably fine.